src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/spreadsheet/PricesSpreadsheetGenerator.kt

```diff
@@ -35,28 +35,26 @@ class PricesSpreadsheetGenerator(
     SXSSFWorkbook().use { workbook ->
       val header = PriceSheet.Header(dateGenerated, DateRange(startDate, endOfMonth(startDate)), supplier)
 
-      val moves = moveService.moves(supplier, startDate)
-
       SummarySheet(workbook, header).also { logger.info("Adding summaries.") }
         .apply { writeSummaries(moveService.moveTypeSummaries(supplier, startDate)) }
 
       StandardMovesSheet(workbook, header).also { logger.info("Adding standard prices.") }
-        .apply { moves[MoveType.STANDARD]?.let { writeMoves(it) } }
+        .apply { writeMoves(moveService.movesForMoveType(supplier, MoveType.STANDARD, startDate)) }
```

Comment:
Decided to load the moves at the point at which they are required for each individual spreadsheet tab in an effort to prevent loading all into memory in one go.